### PR TITLE
Modal: Component Added

### PR DESCRIPTION
**Pull request by m26unkwn**
Modal Component Added for the desktop

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]
Everything is good to go.

1. Clean UI
2. Code is readable avoid !importent

File: components/modal.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      modalBtn.addEventListener("click", () => {
+        modal.classList.add("active");
+      });
+      closeModal_2.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
+      closeModal.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
+    </script>
```
Comment (manojsarna) [inline components/modal.html:207]: Suggestion : the JS Code in a separate file for code reusability and separation of the logic from design

**Review by manojsarna — APPROVED** [review]
Overall code is good

Do check the comments

File: components/modal.html (hunk 1 above)
Comment (m26unkwn) [reply components/modal.html:207]: I am just using it here coz it is a small piece so that why I am keeping it here.

**Pull request merged**